libs/features/webapp-prompt/webapp-prompt.js

```diff
@@ -108,8 +111,8 @@ class AppPrompt {
 
   // TODO: should we allow for app icon to be set as SVG?
   decorate = () => {
-    this.elements.closeIcon = toFragment`<button class="appPrompt-close"></button>`;
-    this.elements.cta = toFragment`<button class="appPrompt-cta appPrompt-cta--close">${this.cancelText}</button>`;
+    this.elements.closeIcon = toFragment`<button daa-ll="Close Modal" class="appPrompt-close"></button>`;
+    this.elements.cta = toFragment`<button daa-ll="${this.cancelText}" class="appPrompt-cta appPrompt-cta--close">${this.cancelText}</button>`;
```

Comment:
What is special about those attributes, we usually used the this [utility](https://github.com/adobecom/milo/blob/stage/libs/blocks/global-navigation/utilities/utilities.js#L124) to process any analytics values

Reply:
We already use `toFragment` - also the microbundle would just use that specific function and tree-shake the rest, so it's safe to use

Reply:
Changed to static values so I don't think we need that utility function anymore. From what I saw, the purpose of that was to ensure no invalid characters are present, which could be the case for dynamic values, but not for static.